WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -97,4 +105,26 @@ public void onJSONException(JSONException exception) {
             }
         });
     }
+
+    @Override
+    public void onPersonSelected(Person person) {
+        FragmentManager fragmentManager = getFragmentManager();
+        PersonDetailFragment personDetailFragment = (PersonDetailFragment) fragmentManager.findFragmentByTag(KEY_PERSON_DETAIL_FRAGMENT);
+
+        long personID = person.getPersonID();
+        int localTableBlogID = person.getLocalTableBlogId();
+        if (personDetailFragment == null) {
+            personDetailFragment = PersonDetailFragment.newInstance(personID, localTableBlogID);
+        } else {
+            personDetailFragment.setPersonDetails(personID, localTableBlogID);
+        }
+        if (!personDetailFragment.isAdded()) {
+            FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
+            fragmentTransaction.setCustomAnimations(R.animator.fragment_slide_in_from_right, R.animator.fragment_slide_out_to_left,
```

Comment:
I feel like being consistent with the other animations is more important here. Since from user's perspective, there is no difference if these are activities or fragments, consistent navigation animations will feel better for them.

The issue with the rotation only occurs if you go into the detail page, rotate and come back. I can look up if we can fix this, but even if it stays, I don't think it's _that_ big of a deal. All in all, I am for keeping it the way it is now.

Having said all of that, I think this goes beyond our personal preferences and affects the design and I think deserves input from more people. So, let's ping @rickybanister, @maxme, @nbradbury & @mattmiklic to get their opinion.